README.md

```diff
@@ -1 +1,116 @@
+<p align="center">
+<a href="https://stormbit.finance"><img src="./docs/StormbitLogo.png" alt="Stormbit Logo" width="120"/></a>
+</p>
 
+
+
+![Foundry](https://img.shields.io/badge/contracts%20built%20with-Foundry-purple)
+![ScaffoldETH](https://img.shields.io/badge/monorepo%20on%20-scaffoldETH-purple)
+![ScaffoldETH](https://img.shields.io/badge/monorepo%20on%20-scaffoldETH-purple)
+
+![Twitter Follow](https://img.shields.io/twitter/follow/StormbitX?style=social)
+
+## Stormbit: Empowering Inclusive Finance
+
+Stormbit is an innovative peer-to-peer lending platform designed to bridge traditional and crypto financial systems, fostering inclusivity and offering superior yields through custom loans. Leveraging blockchain technology, we provide undercollateralized, NFT-backed, and personalized lending options, aiming to democratize access to financial services.
+
+### What Sets Stormbit Apart?
+
+- **Inclusive Lending**: Committed to providing financial services access to underserved populations through the power of decentralized finance (DeFi).
+- **Custom Loans**: Offering the freedom for users to tailor loans according to their specific needs, ensuring flexibility and control.
+- **Superior Yields**: Competitive yields for lenders, significantly outpacing traditional banking returns.
+
+### Getting Started with Stormbit
+
+For detailed instructions on how to engage with the Stormbit platform, including installation and setup, please refer to our [Documentation](https://stormbit.finance/docs).
+
+
+
+## Setting Up Your Development Environment
+
+Before you begin, ensure you have Node.js installed on your system. This project requires Node version 18.17 or later.
+
+
+### Clone the Repository
+
+First, clone the repository to your local machine:
+
+```bash
+git clone https://github.com/Quantum3-Labs/Stormbit-monorepo.git
+cd Stormbit-monorepo
+``` 
+
+
+### Install Dependencies
+
+To install the required dependencies, run:
+
+```bash
+yarn install 
+``` 
+
+### Managing Environment Variables
+
+To configure environment variables for development and production:
+
+
+```bash 
+# Alchemy API Key
+NEXT_PUBLIC_ALCHEMY_API_KEY=<your_alchemy_api_key_here>
+
+# WalletConnect Project ID
+NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=<your_wallet_connect_project_id_here>
+``` 
+
+
+## Running the Application
+
+
+### To start the development server, run:
```

Comment:
I think here we have to specify to go to `packages/nextjs` and then run `yarn install && yarn dev`.
Or we should add the command in root `package.json `

Reply:
done